assets/installer/linkerd-job.yaml

```diff
@@ -0,0 +1,59 @@
+---
```

Comment:
Should these installer assets still be in the repo ?

Reply:
We decided to keep a version of the assets and the manifest on this repo for CI use (to avoid chicken-egg problems with the helm chart repository). But I think I can get rid of the `manifest-small.yaml` file. It shouldn't be needed anywhere.

Reply:
removed the small manifest